content/docs/2.9/scalers/azure-event-hub.md

```diff
@@ -61,13 +61,26 @@ triggers:
 > 💡 Learn more about the checkpointing behaviour in this [section](#checkpointing-behaviour).
 
 > 💡 The Azure Storage connection string is not compatible with connection string created from a Shared Access Signature.
+
 ### Authentication Parameters
 
-The common way of authenticating to Azure Event Hub is by using the connection string. However, you can use [Azure AD Pod Identity](https://docs.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity) or [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/) providers if you host your cluster in Azure AKS, and if have configured it to support Pod Identity.
+You can authenticate by using pod identity or connection string authentication.
 
-To use Pod Identity, you have to add a [TriggerAuthentication](../concepts/authentication.md#re-use-credentials-and-delegate-auth-with-triggerauthentication) and configure it to use Pod Identity like so:
+**Connection String Authentication:**
 
-```
+- `connection` - Connection string for the Azure Service Bus Namespace.
+  
+  The following formats are supported.
+  
+  - With **SharedAccessKey** - 
+    `Endpoint=sb://<sb>.servicebus.windows.net/;SharedAccessKeyName=<key name>;SharedAccessKey=<key value>`
```

Comment:
Not sure what the best way to document this is. But the connection string can optionally end with EntityPath=<hub-name>. If it doesn't then the eventHubName parameter has to be provided (this has been added in the parameters section).

Reply:
Done, let me know what you think!